deps/readies/bin/getredis5

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python2
+
+import sys
+import os
+import popen2
+import argparse
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "deps/readies"))
+import paella
+
+os.environ["PYTHONWARNINGS"] = 'ignore:DEPRECATION::pip._internal.cli.base_command'
+
+#----------------------------------------------------------------------------------------------
+
+class Redis5Setup(paella.Setup):
+    def __init__(self, nop=False):
+        paella.Setup.__init__(self, nop)
+
+    def common_first(self):
+		pass
+
+    def debian_compat(self):
+		# https://chilts.org/installing-redis-from-chris-leas-ppa/
+        self.run("add-apt-repository -y ppa:chris-lea/redis-server")
+		self.install("redis-server")
```

Comment:
You have extra tabs here, might cause issues in python ...

Reply:
Fixed.